src/LibVLCSharp.Forms/Shared/EqualizerUtils.cs

```diff
@@ -0,0 +1,192 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using LibVLCSharp.Shared;
+using Xamarin.Forms;
+
+namespace LibVLCSharp.Forms.Shared
+{
+    /// <summary>
+    /// Defines the <see cref="EqualizerUtils" />.
+    /// </summary>
+    public static class EqualizerUtils
+    {
+        /// <summary>
+        /// Defines the PRECISION.
+        /// </summary>
+        private const float PRECISION = 10;
+
+        /// <summary>
+        /// Defines the RANGE.
+        /// </summary>
+        private const float RANGE = 200;
+
+        /// <summary>
+        /// Defines the EqualizerEnablePropertyKey.
+        /// </summary>
+        private const string EqualizerEnablePropertyKey = "VLC__MediaPlayerElement_IsEqualizerEnable";
+
+        /// <summary>
+        /// Defines the EqualizerPresetIndexPropertyKey.
+        /// </summary>
+        private const string EqualizerPresetIndexPropertyKey = "VLC__MediaPlayerElement_EqualizerPresetIndex";
+
+        /// <summary>
+        /// Load all Presets.
+        /// </summary>
+        /// <returns>The <see cref="List{Preset}"/>.</returns>
+        public static List<Preset> LoadAllPresets()
+        {
+            var presetCount = new Equalizer().PresetCount;
```

Comment:
you want to be using `using` here, as `Equalizer` is a native resource that needs to be released.